setup.py

```diff
@@ -14,8 +19,9 @@
     author_email='info@scrapinghub.com',
     url='http://github.com/scrapinghub/python-scrapinghub',
     platforms = ['Any'],
-    py_modules = ['scrapinghub'],
-    install_requires = ['requests'],
+    py_modules = ['scrapinghub', 'hubstorage'],
```

Comment:
> from scrapinghub import ... should work as before (I think it's only APIError and Connection

btw, ^ my fault, for backwards compatibility we need everything from `legacy.py` to be importable from `scrapinghub`. So you need to move `__all__ = ["APIError", "Connection", "HubstorageClient"]` from `scrapinghub/legacy.py` to `scrapinghub/__init__.py`, and add `from .legacy import *` to `scrapinghub/__init__.py`. If someone has `from scrapinghub import Project` their code will be broken by current change.